### ISSUE TITLE
Readme example fails to run

### ISSUE THREAD

**Issue by ss-remygreinhofer** [NerdWalletOSS/terraformpy#52]
Toying around with `terraformpy`, I attempted to run the example in the `README.md` file, but I was not able to completed the process:

`main.tf.py`:
```python
from terraformpy import Provider, Resource

Provider("aws", profile="default", region="us-east-1")

Resource("aws_instance", "example", ami="ami-2757f631", instance_type="t2.micro")
```

output:
```bash
$ terraformpy
terraformpy - Processing: main.tf.py
Traceback (most recent call last):
  File "/Users/remygreinhofer/remy-project/labs/terraformpy-test-project/venv/bin/terraformpy", line 11, in <module>
    load_entry_point('terraformpy', 'console_scripts', 'terraformpy')()
  File "/Users/remygreinhofer/remy-project/labs/terraformpy-test-project/vendor/terraformpy/src/terraformpy/cli.py", line 42, in main
    imp.load_source(filename[:-6], filename)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "main.tf.py", line 3, in <module>
    Provider("aws", profile="default", region="us-east-1")
  File "/Users/remygreinhofer/remy-project/labs/terraformpy-test-project/vendor/terraformpy/src/terraformpy/objects.py", line 89, in __new__
    inst = super(TFObject, cls).__new__(cls, *args, **kwargs)
TypeError: object.__new__() takes exactly one argument (the type to instantiate)
```

This happened using `terraformpy 1.1.2`.

**Comment by ss-remygreinhofer** [comment]
I just noticed that `terraformpy` seems to be for Python 2 only. That's probably why the example did not work.

**Comment by borgstrom** [comment]
This will be fixed in #54
